BMM.Core/ViewModels/AlbumViewModel.cs

```diff
@@ -143,7 +154,17 @@ private void UpdateView(object sender, NotifyCollectionChangedEventArgs e)
 
         public bool ShowFollowButtons => false;
 
-        public bool ShowShuffleButton => Documents.OfType<Track>().Any();
+        public bool ShowShuffleOrResumeButton => Documents.OfType<Track>().Any();
+        public string ShowShuffleOrResumeText => GetShowShuffleOrResumeText();
+
+        private string GetShowShuffleOrResumeText()
```

Comment:
As mentioned before, why not have 2 buttons?

Reply:
Another reason for having 2 buttons, is that I could imagine that we show both buttons in the future.